tests/zone.test.ts

```diff
@@ -0,0 +1,119 @@
+import { getDefaultAddress, getAddresses, addAddress, removeAddress, removeCoin, setAddresses } from "../src/zone";
+import { parseZoneFile, makeZoneFile } from "../src/zone";
+
+const file1 = `$ORIGIN muneeb.id
+$TTL 3600
+_http._tcp URI 10 1 "https://gaia.blockstack.org/hub/1J3PUxY5uDShUnHRrMyU6yKtoHEUPhKULs/0/profile.json"
+`
+const file2 = `$ORIGIN muneeb.id
+$TTL 3600
+_http._tcp URI 10 1 "https://gaia.blockstack.org/hub/1J3PUxY5uDShUnHRrMyU6yKtoHEUPhKULs/0/profile.json"
+_btc._addr.muneeb.id. IN TXT "0x123"
```

Comment:
Might be good to replace the 0x123 with an actual BTC or STX address like:
`1JxoezfGM7XgRTZAsZHwamLVHrT5RQLuHQ` and `SP32GFXJ01KFS4XRA718XPYP9TBWZP3J63RVT3JAB`

Would be good to add an Eth example as well.